spec.bs

```diff
@@ -387,22 +387,23 @@ The <dfn for=Navigator method>runAdAuction(|config|)</dfn> method steps are:
   1. If |signal| is [=AbortSignal/aborted=], then [=reject=] |p| with |signal|'s
     [=AbortSignal/abort reason=] and return |p|.
   1. [=AbortSignal/Add|Add the following abort steps=] to |signal|:
-    1. TODO: Specify. Abort the auction. If the auction has already completed, this has no
-      effect. Otherwise, stop running new worklet functions, and do not cause side-effects, like
-      reporting and updating bid statistics.
     1. [=Reject=] |p| with |signal|’s [=AbortSignal/abort reason=].
+    1. Run [=interest group update=] with |auctionConfig|'s
+      [=auction config/interest group buyers=].

```

Comment:
Maybe follow-up work: according to https://pr-preview.s3.amazonaws.com/qingxinwu/turtledove/pull/540.html#auction-config-interest-group-buyers - interest group buyers can be null. Does `[=interest group update=]` handle that correctly?

Reply:
when interest group buyers are null, [[=interest group update=]](https://pr-preview.s3.amazonaws.com/qingxinwu/turtledove/pull/540.html#interest-group-update) does nothing. It seems the correct thing to do. Or am I missing something?

Reply:
I guess this comes down to whether https://infra.spec.whatwg.org/#list-iterate handles null okay which is unclear to me, but I imagine it's fine.